neo/bin/prompt.py

```diff
@@ -142,6 +142,7 @@ class PromptInterface:
                 'wallet split {addr} {asset} {unspent index} {divide into number of vins}',
                 'wallet close',
                 'send {assetId or name} {address} {amount} (--from-addr={addr}) (--fee={priority_fee})',
+                'sendmany (--outgoing={number outgoing}) (--change-addr={addr}) (--from-addr={addr}) (--fee={priority_fee})',
```

Comment:
we have to think of a clearer way to explain this function. I had to read the code to understand that `outgoing` is a number that collects the `outputs_array` of the [C# description](http://docs.neo.org/en-us/node/cli/2.7.6/api/sendmany.html). For command arguments placing something between `()` parentheses means it's an optional argument. I'm pretty sure that's not the case here. I also can't get it to work:

```
neo> sendmany 0 neo neo
Asset id not found
neo> sendmany 1 neo neo
Asset id not found
neo> sendmany 1 neo
Not enough arguments
neo> sendmany 1 neo wut
Asset id not found
neo> sendmany
Not enough arguments
neo> sendmany --outgoing=0 neo bla
Asset id not found
```

Reply:
--outgoing is the only required command because it sets up how many iterations (transactions) prompt will ask for. I will update with what might be a better description. The test `send_many_good_simple` illustrates how the function should work https://github.com/jseagrave21/neo-python/blob/ee6c37c9aa2b4ef3c39977e469f5057d96dc1dce/neo/Prompt/Commands/tests/test_send_command.py#L336